webapp/src/components/sidebar_right/index.tsx

```diff
@@ -51,31 +51,51 @@ export const renderThumbVertical = (props: Props) => (
     />
 );
 
+// React hook to store the last state of "yourPrs" and "reviews"
+function usePrevious(value: Item[]) {
```

Comment:
Shouldn't we make this hook generic so that it can also be used in other components if required?